pulsar-broker/src/test/java/org/apache/pulsar/client/api/SubscriptionMessageDispatchThrottlingTest.java

```diff
@@ -220,6 +220,93 @@ public void testMessageRateLimitingReceiveAllMessagesAfterThrottling(Subscriptio
         log.info("-- Exiting {} test --", methodName);
     }
 
+    @Test(dataProvider = "subscriptions", timeOut = 30000)
```

Comment:
You can add invocationCount on the Test annotation to call the new test multiple times to understand the problem.

Reply:
@mattisonchao If the test can't reproduce the issue stable, we can also add the invocationCount to make the test can get passed multiple times on the CI environment.

Reply:
Sure